src/content/updates/whats-new/whats-new.mdx

```diff
@@ -1,28 +1,223 @@
+
+- Theming enhancements
+- Icon updates
+- UI shell enhancements
+- Support and component enhancements
+- Component specification
+- New contribution guidelines
+
+We also introduced general updates to our website and have an ongoing support project for managing issue triage and fixing bugs.
+
+### Theming enhancements
+
+#### Enhancements 🛠 
+
+- Added color theme dropdown in vanilla development environment (https://the-carbon-components.netlify.com) https://github.com/carbon-design-system/carbon/pull/3073
+- Updated theming sandbox with v10 components and color themes (http://themes.carbondesignsystem.com)
+
+#### Bug fixes 🐛 
+
+- Button disabled state background color in dark themes
+- Ghost button text/icon colors in dark themes
+- Danger button border color in dark themes
+- Disabled checkbox to show check in dark themes
+- Content switcher to use `$disabled-0*` color tokens for dark themes
+- Inline dropdown background and disabled colors in dark themes
+- Skeleton color tokens to work in all themes
+- Theme `$highlight` token in dark themes
+- Theme `$hover-primary-text` token in dark themes
+- Theme `$hover-ui` token in g90 theme
+
+#### Features:
+
+- Emit custom properties for `@carbon/themes` color tokens
+
+### Icon updates
+
+#### Bug fixes 🐛 
+
+* battery--low.svg
+* ibm-security.svg
+* logo--slack.svg
+* logout.svg
+* mobile--add.svg
+* mobile--check.svg
+* mobile--download.svg
+* mobile.svg
+* popup.svg
+* text--strikethrough.svg
+* mammogram.svg
+
+#### Enhancements 🛠 
+
+- Bespoke icons added at the following sizes: `20/time.svg`
+- `@carbon/icons-react` now supports an entrypoint `index.js`, you no longer are required to specify the path for an icon
+
+#### New ✨ 
+
+* language.svg
+* laptop.svg
+* account.svg
+* branch.svg
+* enterprise.svg
+* fog.svg
+* help-desk.svg
+* no-image.svg
+* no-ticket.svg
+* pause--filled.svg
+* play--filled--alt.svg
+* service-desk.svg
+* skip--back--filled.svg
+* skip--forward--filled.svg
+* stop--filled--alt.svg
+* audio-console.svg
+* battery--empty.svg
+* battery--quarter.svg
+* carbon.svg
+* category--add.svg
+* category--and.svg
+* category--new-each.svg
+* category--new.svg
+* category.svg
+* concept.svg
+* fork.svg
+* hurricane.svg
+* mobile--landscape.svg
+* partly-cloudy.svg
+* sunny.svg
+* sunset.svg
+* tablet--landscape.svg
+* tablet.svg
+* term.svg
+* text-link--analysis.svg
+* text-link.svg
+* text-mining--applier.svg
+* text-mining.svg
+* tree-view--alt.svg
+* type-pattern.svg
+* types.svg
+
+#### Removed ☠️ 
+
+* mammogram--stacked.svg
+
+### UI shell enhancements
+
+#### New ✨ 
+
+- Guidance for UI Shell Navigation patterns
+- UI Shell Sketch template and symbols added to the Sketch kit.
+- Rail functionality for UI shell left panel.
+- Responsive behavior for the header with nav links.
+
+### Support and component enhancements
+
+#### Bug fixes 🐛 
+
+- Accessibility updates to Tooltips
+
+   * Definition tooltip WCAG 2.1 compliance (#3151)
```

Comment:
In all the above bullet points, we don't leave a space between each bullet point. Should there be a space above here? Also, we're not linking any urls for the PRs here, like in the Design Kit bullet points, should we just remove the PR number then?

Reply:
Our nested unordered lists are broken on the site rn. It's not broken on the Gatsby theme so I figured it's not an emergency until we migrate. I can remove the PR number, I just pulled this from GH and it looks like there wasn't an actual link there.